jest.config.ts

```diff
@@ -41,8 +41,14 @@ export default {
   // A list of reporter names that Jest uses when writing coverage reports
   coverageReporters: ["json", "text", "text-summary", "lcov"],
 
-  // An object that configures minimum threshold enforcement for coverage results
-  // coverageThreshold: undefined,
+  // TODO: Increase threshold as additional tests are added
+  coverageThreshold: {
+    global: {
+      branches: 18.3,
+      functions: 9.05,
+      statements: 9.44
+    }
+  },
```

Comment:
my preferred alternative to enforcing coverage in jest, is to do it with codecov (which can merge coverage reports across multiple jobs).

Reply:
Definitely want to replace with something more robust. Having this in as a quick stopgap.